plugin/src/main/java/net/aufdemrand/denizen/events/entity/EntityCombustsScriptEvent.java

```diff
@@ -108,6 +113,20 @@ public dObject getContext(String name) {
         else if (name.equals("duration")) {
             return new Duration(burntime);
         }
+        else if (name.equals("source")) {

```

Comment:
i would still like to see the CombustByBlock source here and a removal of the note in the docs

Reply:
We talked about this in discord, but just to note here - This always returning null is a bug that may be fixed in the future. Because of this, we should preemptively implement it and just note in the docs that it currently always returns null.
https://hub.spigotmc.org/stash/projects/SPIGOT/repos/craftbukkit/browse/nms-patches/Entity.patch#198
https://hub.spigotmc.org/stash/projects/SPIGOT/repos/craftbukkit/browse/nms-patches/Entity.patch#265